xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp

```diff
@@ -733,6 +734,18 @@ bool CVideoPlayerVideo::ProcessDecoderOutput(double &frametime, double &pts)
     if (m_speed != 0)
       pts += m_picture.iDuration * m_speed / abs(m_speed);
 
+    // ATSC A53 Closed Captions
+    if (m_picture.HasA53SideData() &&
+        CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(
+            CSettings::SETTING_SUBTITLES_PARSECAPTIONS))
+    {
+      auto a53SideData = m_picture.ConsumeA53SideData();
+      std::unique_ptr<CCaptionBlock> ccData =
```

Comment:
This is however a good place to use auto as you know it will be `std::unique_ptr<CCaptionBlock>` from the assignment.
```suggestion
      auto ccData =
```